app/serializers/rest/follow_requested_account_serializer.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class REST::FollowRequestedAccountSerializer < REST::AccountSerializer
+  attribute :follow_request_created_at
+
+  def follow_request_created_at
+    FollowRequest.find_by(account_id: object.id).created_at
```

Comment:
Does this necessarily find the correct record?

At the controller level there's a query which makes sure that the accounts which are serialized are the accounts where the `current_account` (signed in user) is the target of the request ... but I think in this serializer the way this finder is now it would just find the first follow request where the account being serialized is the requester, which would not necessarily be the one relevant to current_account ... ?

We may need to pass in the current_account to serializer in order to confidently get the correct one.